homeassistant/components/homekit/__init__.py

```diff
@@ -214,6 +217,10 @@ def add_bridge_accessory(self, state):
         """Try adding accessory to bridge if configured beforehand."""
         if not state or not self._filter(state.entity_id):
             return
+        self.devices_added += 1
+        if self.devices_added == MAX_DEVICES + 1:
```

Comment:
Don't add the check here. This will get checked for every accessory. So a worst case could be for 200 entities: 100 warnings. Instead insert this in the `start` method above:
```py
# Insert here

_LOGGER.debug('Driver start')
self.hass.add_job(self.driver.start)
self.status = STATUS_RUNNING
```

Reply:
Also `> MAX_DEVICES` should be enough.

Reply:
thats why i used `==` . So you will get the warning only once. But you are right, using `len()` is ways better.

Reply:
You're right. Also just the point that I though it could be a problem shows the statement and its position wasn't optimal ;)